### ISSUE TITLE
Build docker container

### ISSUE THREAD

**Issue by quartermaine** [HealisticEngineer/Docker#5]
I am trying to build the container using the command provided 

`docker build -t healisticengineer/pbirs1.14 .`
but I am getting an error 

`ERROR: failed to solve: mcr.microsoft.com/windows/servercore:ltsc2019: no match for platform in manifest sha256:3cba12bffca83466997158c8a0d38f20a18fb14f5ec62cd6454dfdd328278d87: not found`

**Comment by DDSNA** [comment]
Hello, I know I'm super late to the party but that is mainly happening because of architecture differences. If you'd try to run this on an ARM processor such as a RaspberryPi, it most likely would lead to this error as the software this project discusses requires x64 architecture => it's probably the wrong architecture for that use case.
I don't think there's an easy solution to this issue, as it's also a software dependency issue - not just a docker image configuration problem.

Dropping this solely for documentation purposes of any future internet searchers.

**Comment by HealisticEngineer** [comment]
Yes, as there is no support for non-x64 for PowerBI Report Server.
If you where able to load the windows using another image it would then fail on the powerbi install.
